src/app/view/perspectives/post-it/post-it-perspective.component.ts

```diff
@@ -273,6 +276,10 @@ export class PostItPerspectiveComponent implements OnInit, OnDestroy {
   }
 
   private addDocumentsToLayoutAndGetTheirCollections(documents: Document[]): void {
+    if (documents.length === 0) {
+      this.allLoaded = true;
```

Comment:
This will introduce some bugs. You are setting `allLoaded` only when an empty list of documents is received but checking it always in the condition a few lines above.

Reply:
What bugs? Once the server returns an empty page, all pages are already loaded so the client stops requesting more.

Reply:
I see. But we should start thinking in terms of reactive components since we are gradually rewriting components using NgRx. If we ignore it until refactoring then the complete rewrite of a component will be needed.